src/transport/FabricTable.cpp

```diff
@@ -541,6 +526,15 @@ CHIP_ERROR FabricTable::Delete(FabricIndex id)
     return CHIP_NO_ERROR;
 }
 
+void FabricTable::DeleteAllFabrics()
+{
+    static_assert(CHIP_CONFIG_MAX_DEVICE_ADMINS + kMinValidFabricIndex <= UINT8_MAX, "Cannot create more fabrics than UINT8_MAX");
+    for (FabricIndex i = kMinValidFabricIndex; i < (CHIP_CONFIG_MAX_DEVICE_ADMINS + kMinValidFabricIndex); i++)
```

Comment:
As a followup, the fact that we have this nontrivial loop condition in multiple places is not great.  Should think of a way to make it less error-prone.

Reply:
Issue: https://github.com/project-chip/connectedhomeip/issues/8920